### ISSUE TITLE
multiplication issue

### ISSUE THREAD

**Issue by chinmayghegde** [HareInWeed/gec#4]
I configured a test file as mentioned in the read me. when multiplied with 2 or 8 etc the result is. may be the point doubling is having issue,

{0x0000000000000000 0000000000000000 0000000000000000 0000000000000000,
 0x0000000000000000 0000000000000000 0000000000000000 0000000000000000,
 0x0000000000000000 0000000000000000 0000000000000000 0000000000000000} 

In addition i am really trying to understand few parts of parameters eg.

const Secp256k1 GEN(
    Field(0x9981e643e9089f48979f48c033fd129c231e295329bc66dbd7362e5a487e2097_int),
    Field(0xcf3f851fd4a582d670b6b59aac19c1368dfc5d5d1f1dc64db15ea6d2d3dbabe2_int),
    Field(0x1000003d1_int),
and 
const Field B(0x700001ab7_int);  // = B * 2^256 mod MOD

not sure if its a different way of defining the parameters and documentation around this would be helpful for the users.

**Comment by loganercanbrack** [comment]
Yeah in the same boat. The 0 issue is when you don't. perform all of the kernel functions
